app/views/hyrax/homepage/index.html.erb

```diff
@@ -6,7 +6,11 @@
 
 <div class="home-content">
   <%= render '/adl/about_section' %>
-  <%= render '/adl/featured_collections' %>
+  <%# this conditional hides featured collections in non-adl tenants. fill in your tenant's name, and switch the comments on the following 2 lines to see this section in the development environment %>
+  <%# if current_account.cname == "<YOUR TENANT NAME HERE>" %>
+  <% if current_account.name == "adl" %>
+    <%= render '/adl/featured_collections' %>
+  <% end %>
   <%= render '/adl/featured_works' %>
```

Comment:
Should this be in the `<% if current_account.name == "adl" %>` block?

Reply:
the comment says we only want to hide featured collections for tenants that aren't adl, so yes I think this placement is correct.

Reply:
Are you asking about the featured_works? If not, then no, Katharine only wants to hide the collections (she recently changed her mind about this and told me today in a different ticket)